tutorials/R/R-skills/Colorado-floods-data-visualization/COOP-NEIS-Precipitation-In-R/COOP-NEIS-Precipitation-In-R.md

```diff
@@ -596,12 +597,13 @@ As an added challenge, aggregate the data by month instead of by day.
 
     ## PhantomJS not found. You can install it with webshot::install_phantomjs(). If it is installed, please make sure the phantomjs executable can be found via the PATH variable.
```

Comment:
I think that I was able to fix this by installing PhantomJS with the install line included.